testapi.pm

```diff
@@ -921,6 +921,10 @@ Use C<script_run> instead if C<$cmd> may fail.
 
 C<$fail_message> is returned in the die message if specified.
 
+C<$quiet> and C<$max_interval> are passed through to C<< $distri->script_run >>. In
```

Comment:
s/C<< $distri->script_run >>/C<$distri->script_run>/

Reply:
The double `<<>>` are needed because of the inner `>`